FreydCategoriesForCAP/gap/precompiled_categories/CategoryOfColumnsAsOppositeOfCategoryOfRowsOfFieldPrecompiled.gi

```diff
@@ -34,16 +34,29 @@ end
         
 ########
 function ( cat_1, arg2_1, arg3_1 )
-    local hoisted_1_1, hoisted_2_1, hoisted_3_1, deduped_4_1, deduped_5_1, deduped_6_1;
-    deduped_6_1 := RankOfObject( arg2_1 );
-    deduped_5_1 := RankOfObject( arg3_1 );
-    deduped_4_1 := deduped_5_1 * deduped_6_1;
-    hoisted_3_1 := deduped_6_1;
-    hoisted_2_1 := deduped_5_1;
-    hoisted_1_1 := HomalgIdentityMatrix( deduped_4_1, UnderlyingRing( cat_1 ) );
-    return List( [ 1 .. deduped_4_1 ], function ( logic_new_func_x_2 )
+    local hoisted_1_1, hoisted_2_1, hoisted_3_1, hoisted_4_1, hoisted_5_1, hoisted_6_1, hoisted_7_1, hoisted_8_1, hoisted_9_1, deduped_10_1, deduped_11_1, deduped_12_1, deduped_13_1, deduped_14_1, deduped_15_1;
+    deduped_15_1 := ModelingCategory( cat_1 );
+    deduped_14_1 := Opposite( RankOfObject( arg2_1 ) );
+    deduped_13_1 := Opposite( RankOfObject( arg3_1 ) );
+    deduped_12_1 := RankOfObject( deduped_14_1 );
+    deduped_11_1 := RankOfObject( deduped_13_1 );
+    deduped_10_1 := deduped_11_1 * deduped_12_1;
+    hoisted_9_1 := ObjectifyObjectForCAPWithAttributes( rec(
+           ), deduped_15_1, Opposite, deduped_13_1 );
+    hoisted_8_1 := ObjectifyObjectForCAPWithAttributes( rec(
+           ), deduped_15_1, Opposite, deduped_14_1 );
+    hoisted_7_1 := deduped_15_1;
+    hoisted_6_1 := deduped_14_1;
+    hoisted_5_1 := deduped_13_1;
+    hoisted_4_1 := OppositeCategory( deduped_15_1 );
+    hoisted_3_1 := deduped_12_1;
+    hoisted_2_1 := deduped_11_1;
+    hoisted_1_1 := HomalgIdentityMatrix( deduped_10_1, UnderlyingRing( cat_1 ) );
+    return List( [ 1 .. deduped_10_1 ], function ( logic_new_func_x_2 )
             return ObjectifyMorphismWithSourceAndRangeForCAPWithAttributes( rec(
-                   ), cat_1, arg2_1, arg3_1, UnderlyingMatrix, ConvertRowToMatrix( CertainRows( hoisted_1_1, [ logic_new_func_x_2 ] ), hoisted_2_1, hoisted_3_1 ) );
+                   ), cat_1, arg2_1, arg3_1, UnderlyingMatrix, ObjectifyMorphismWithSourceAndRangeForCAPWithAttributes( rec(
```

Comment:
This compiled line looks wrong.

Reply:
Yes, `BasisOfExternalHom` and `Coefficients...` are currently broken.